que-web.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "que", "~> 0.8"
   spec.add_dependency "sinatra"
   spec.add_dependency "erubis"
+  spec.add_dependency "activesupport"
```

Comment:
I lean more in favor of dropping the activesupport dependency. It's a lot to bring on for just `#present?`. I didn't catch the usage when reviewing the search feature pull request.